backend/Node/routes/openWeatherRoute.ts

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import { currentWeatherData } from './route-helpers/openweather-helpers';
+import { isLatitude, isLongitude } from './route-helpers/mapbox-helpers';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    res.send("Open Weather API");
+});
+
+router.get('/reverseWeather/:latLon', async (req, res) => {
+    const latLon = req.params.latLon.split(',');
+    if (!isLatitude(latLon[0]) || !isLongitude(latLon[1])) {
+        res.status(400).send("Invalid lat/lon value(s)");
```

Comment:
Let's stick with single quotes since we're using single quotes in other parts of the code